src/test/test-copy.c

```diff
@@ -50,6 +50,76 @@ TEST(copy_file) {
         unlink(fn_copy);
 }
 
+static bool read_file_and_streq(const char* filepath, const char* expected_contents) {
+        _cleanup_free_ char *buf = NULL;
+        if (read_full_file(filepath, &buf, NULL) != 0)
```

Comment:
Missing newline between declarations and first statement

Reply:
Also, you can just use `assert_se()` here, that makes it easier to figure out what failed if a test fails.